packages/core/src/accessanalyzer/activation.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { ExtContext } from '../shared/extensions'
+import { DevSettings } from '../shared/settings'
+import { Commands } from '../shared/vscode/commands2'
+import { renderPolicyChecks } from './vue/policyChecks'
+
+/**
+ * Activate Policy Checks functionality for the extension.
+ */
+export async function activate(extContext: ExtContext): Promise<void> {
```

Comment:
you will need to call this from the main extension activation. `packages/core/src/extension.ts`